ExamPlatform/Controllers/CandidateController.cs

```diff
@@ -0,0 +1,93 @@
+﻿using ExamPlatform.Models;
+using ExamPlatform.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace ExamPlatform.Controllers
+{
+    public class CandidateController : Controller
+    {
+
+        // Creating instance of Candidate Service
+        CandidateService service = CandidateService.GetInstance;
+
+        
+        // 1. READ OPERATION
+
+        // GET: /Candidate
+        public ActionResult Index()
+        {
+            return View(service.GetCandidateList());
+        }
+        
+        // Get: /Candidate/Details/{id}
+        public ActionResult Details(int id)
+        {
+            return View(service.GetCandidateList().Where(x => x.CandidateId == id).FirstOrDefault());
+        }
+
+        // 2. Create OPERATION
+
+        // GET: /Candidate/Create
+        public ActionResult Create()
```

Comment:
use ValidateAntiForgeryToken also read more about it https://www.c-sharpcorner.com/article/understand-antiforgeri-token-in-asp-net-mvc/